edi_bank_statement_import/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,7 @@
+Plug bank_statement_import into EDI machinery.
+
+
+This module allows to use account_bank_statement_import to parse edi exchange (input).
+This allow you ro retrieve bank stazement from a any edi exchange backend (webservice, sftp, local filesystem, ...)
+
+TODO: shall we add an exchange type example as demo?
```

Comment:
yes pls :)

Reply:
I'm not really able to, but let the message (inspired from #451)

Reply:
Done, please check it

Reply:
See my comment above